src/controllers/still.required.controller.ts

```diff
@@ -1,17 +1,58 @@
 import {NextFunction, Request, Response} from "express";
-import {PTFCompanyProfile} from "../model/company.profile";
+import {check, validationResult} from "express-validator/check";
+import {COMPANY_REQUIRED_NOT_SELECTED} from "../model/error.messages";
+import {createGovUkErrorData, GovUkErrorData} from "../model/govuk.error.data";
 import {STILL_REQUIRED} from "../model/template.paths";
+import {ValidationError} from "../model/validation.error";
 import {getPromiseToFileSessionValue} from "../services/session.service";
 import {COMPANY_PROFILE} from "../session/keys";
 
-export const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+const validators = [
+  check("continuedIllness").not().isEmpty().withMessage(COMPANY_REQUIRED_NOT_SELECTED),
+];
 
-  const company: PTFCompanyProfile = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE);
+export const getRoute = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 
+  const companyName: string = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE).companyName;
+
+  return res.render(STILL_REQUIRED, {
+    companyName,
+    templateName: STILL_REQUIRED,
+  });
+};
+
+const postRoute = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
+  const errors = validationResult(req);
+  const companyName: string = getPromiseToFileSessionValue(req.chSession, COMPANY_PROFILE).companyName;
+
+  // render errors in the view
+  if (!errors.isEmpty()) {
+    const errorText = errors.array()
+      .map((err: ValidationError) => err.msg)
+      .pop() as string;
+
+    return renderPageWithError(res, errorText, companyName);
+  } else {
+    return res.render(STILL_REQUIRED, {
```

Comment:
should this go to the next page after still required?

Reply:
Yes it will but was leaving that open for now